### ISSUE TITLE
Application Direction

### ISSUE THREAD

**Issue by alexmcmillan** [jstogether/jstogether#14]
Application Direction:

(Not concrete, all open for discussion)
1. One project at a time, which everybody works on.
2. Each project will have a single "topic" or "technology" as the main focus for that project
3. Anybody can team up with anybody, but we may set limits on team size (2-3 ppl / solo / 4 ppl)
4. Teams can choose whatever technologies they wish to use, but must adhere to point 2
5. Teams are expected to document their project and its development in some way on their Team page
6. At the end of a project, we have a short break where people can provide feedback and vote on the next project / technology
7. Project solutions can be voted up / down and at the end, each member of a Team whose project received (eg) 40 upvotes receives a "score" of 40, which will be added to their total score and displayed on their Profile

**Comment by jephillips** [comment]
In agreement on 1-6, need further discussion on project scoring. There are both pros and cons of adding in a gamification system to the project. I think designing a proper system is going to be a whole project on its own.

**Comment by alexmcmillan** [comment]
Yeah I envisage 7 will be a far way down the list of priorities, at least for the forseeable future.
